templates/compute/resources/vkcs_compute_interface_attach.md.tmpl

```diff
@@ -12,22 +12,16 @@ description: |-
 
 ## Example Usage
 ### Basic Attachment
```

Comment:
this is not a base case indeed. lets rename it

Reply:
renamed